etc/kayobe/inventory/group_vars/all/firewall

```diff
@@ -0,0 +1,413 @@
+     (stackhpc_storage_firewalld_rules if 'storage' in group_names else []) +
+     (stackhpc_monitoring_firewalld_rules if 'monitoring' in group_names else []) +
+     (stackhpc_seed_firewalld_rules if 'seed' in group_names else []) +
+     (stackhpc_seed_hypervisor_firewalld_rules if 'seed-hypervisor' in group_names else []) +
+     (stackhpc_wazuh_manager_infra_vm_firewalld_rules if 'wazuh-manager' in group_names else []) +
+     (stackhpc_ansible_control_infra_vm_firewalld_rules if inventory_hostname == 'localhost' else []) }}
+
+###############################################################################
+# Common firewalld rules
+
+stackhpc_common_firewalld_rules: "{{ stackhpc_common_firewalld_rules_default + stackhpc_common_firewalld_rules_extra }}"
+
+stackhpc_common_firewalld_rules_default: |
+  {% set stackhpc_common_firewalld_rules_formatted = [] %}
+  {% for ruleset in stackhpc_common_firewalld_rules_template %}
+  {% if ruleset.enabled %}
+  {% for rule in ruleset.rules %}
+  {% set _ = stackhpc_common_firewalld_rules_formatted.append(rule | combine({'zone': rule.network | net_zone })) %}
+  {% endfor %}
+  {% endif %}
+  {% endfor %}
+  {{ stackhpc_common_firewalld_rules_formatted }}
+
+stackhpc_common_firewalld_rules_template:
+  # Common
+  - rules:
+      - service: ssh
+        network: "{{ admin_oc_net_name }}"
+        state: enabled
+    enabled: true
+  - rules:
+      - service: cockpit
+        network: "{{ public_net_name }}"
+        state: disabled
+      - service: dhcpv6-client
+        network: "{{ public_net_name }}"
+        state: disabled
+    enabled: "{{ 'public' in stackhpc_firewalld_zones }}"
+  - rules:
+      - service: ssh
+        network: "{{ public_net_name }}"
+        state: disabled
+    enabled: "{{ 'public' in stackhpc_firewalld_zones && admin_oc_net_name | net_zone != 'public' }}"
+
+stackhpc_common_firewalld_rules_extra: []
+
+###############################################################################
+# Common firewalld rules
+
+stackhpc_controller_firewalld_rules: "{{ stackhpc_controller_firewalld_rules_default + stackhpc_controller_firewalld_rules_extra }}"
+
+stackhpc_controller_firewalld_rules_default: |
+  {% set stackhpc_controller_firewalld_rules_formatted = [] %}
+  {% for ruleset in stackhpc_controller_firewalld_rules_template %}
+  {% if ruleset.enabled %}
+  {% for rule in ruleset.rules %}
+  {% set _ = stackhpc_controller_firewalld_rules_formatted.append(rule | combine({'zone': rule.network | net_zone })) %}
+  {% endfor %}
+  {% endif %}
+  {% endfor %}
+  {{ stackhpc_controller_firewalld_rules_formatted }}
+
+stackhpc_controller_firewalld_rules_template:
+  # Overcloud Ironic
+  - rules:
+      # Ironic inspector API
+      - port: 5050/tcp
+        network: "{{ provision_oc_net_name }}"
+        state: enabled
+      # Ironic API
+      - port: 6385/tcp
+        network: "{{ provision_oc_net_name }}"
+        state: enabled
```

Comment:
I think these are covered by Kolla Ansible's firewall config. They aren't in the RUG config.

Reply:
Yep, fixing the internal network issues resolved that